Soomla/Assets/Plugins/Soomla/Profile/SoomlaProfile.cs

```diff
@@ -882,7 +890,12 @@ public class SoomlaProfile
 		}
 
 		internal static void TryFireProfileInitialized () {
-			if (AllProvidersInitialized()) {
+			if (AllProvidersInitialized()
+#if !UNITY_EDITOR
+				&& nativeModulesInitialized
```

Comment:
@EugeneButusov I'd remove the `#if !UNITY_EDITOR` from here, and simply assign `nativeModulesInitialized = true` somewhere else with `#if UNITY_EDITOR`. I think it will make nicer code.